packages/mephisto-task/src/live.js

```diff
@@ -58,20 +58,17 @@ const useMephistoLiveTask = function (props) {
 
   const { initialTaskData } = taskProps;
 
-  function onMessageReceived(message) {
-    const { state, message_id, timestamp, ...remaining_message } = message;
-    if (state !== undefined) {
-      props.onStateUpdate && props.onStateUpdate({ state });
-    }
+  function handleLiveData(liveData) {
+    const { message_id, timestamp, ...remaining_message } = message;
     if (Object.keys(remaining_message).length != 0) {
```

Comment:
i think we can remove this clause now too right, now that `state` is gone? unless there's a case where people would purposefully send blank liveData payloads?